client/dist/index.html

```diff
@@ -2,9 +2,15 @@
 <html>
   <head>
     <title></title>
+    <link rel="stylesheet" href="attraction-overview.css"></link>
   </head>
   <body>
-    <div id="attractionOverview">
-    <script src="bundle.js"></script>
+    <div id="page-container">
+      <div id="attraction-overview">
```

Comment:
this file should just be a local version, so no need to add other module's divs to this unless you need them for your app.